catalog-manifests/dedicated-admin-operator/dedicated-admin-operator.package.yaml

```diff
@@ -0,0 +1,4 @@
+channels:
+- currentCSV: dedicated-admin-operator.v0.1.67-2019-04-09-091172db
+  name: generalize-builds
```

Comment:
I imagine this channel name should be renamed to `master` when merging.

Reply:
This is generated, so it will be changed when we merge.

Reply:
I have reverted `catalog-manifests/` to what it was prior to updates, this one generated version is gone and the channel name reset to what it was, `production`. Subsequent builds will set it based on the branch.